pom.xml

```diff
@@ -733,6 +711,41 @@
 
     <profiles>
         <profile>
+            <id>qa</id>
+            <activation>
+                <property>
+                    <name>skipQA</name>
+                    <value>!true</value>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <version>2.12.1</version>
```

Comment:
I'd prefer to leave the configuration of the plug-in version in the main pluginManagement block. That way the plug-in e.g. can be skipped for single modules if needed, without the warning about the missing version. I.e. like so:

    <build>
        <pluginManagement>
            <plugins>
                <plugin>
                    <artifactId>maven-checkstyle-plugin</artifactId>
                    <version>2.12.1</version>
                </plugin>
        ...

    <profiles>
        <profile>
            <id>qa</id>
            ...
            <build>
                <plugins>
                   <plugin>
                       <artifactId>maven-checkstyle-plugin</artifactId>
                       <configuration>...</configuration>
                       <executions>
                    ...